### PR TITLE
Initial commit for meetup changes

### PR DESCRIPTION
**Pull request by neelab**
Reverse order of events.
Change URL.

### CODE REVIEW

File: konfera/register/urls.py [hunk 1]
```diff
@@ -20,5 +20,5 @@
 
 urlpatterns = [
     url(r'^volunteer/event/(?P<slug>[\w, -]+)/$', views.register_volunteer, name='event_register_volunteer'),
-    url(r'^meetups/', views.meetup_list, name='list_all_meetups'),
+    # url(r'^meetups/', views.meetup_list, name='list_all_meetups'),
```
Comment (ricco386) [inline konfera/register/urls.py]: no need to keep this line commented (as the url has moved), please remove.

File: konfera/urls.py [hunk 2]
```diff
@@ -19,4 +19,5 @@
 urlpatterns = [
     url(r'^events/', include('konfera.event.urls'), name='event_list'),
     url(r'^register/', include('konfera.register.urls')),
+    url(r'^meetups/', 'views.meetup_list', name='list_all_meetups'),
```
Comment (ricco386) [inline konfera/urls.py]: this is referencing to non existing view. Please create views.py in the konfera directory and move the view logic from register.views function `meetup_list` into newly created views.

Comment (neelab) [inline konfera/urls.py]: Thanks for the feedback! I'll definitely go back and update - I saw the Travis tests had failed. Once I am able to run the app I'll be able to fix and test, just wanted to get an early PR out there.

**Comment by neelab** [comment]
@ricco386 - See changes above. Do I need to add in tests for the codecov/patch percentage to increase?

**Comment by ricco386** [comment]
@neelab your python code looks good to me so it is ready to be merged. However in the but there is also problem with template please update those as well...

* pagination is not styled (see events list how the pagination shoul looks like).
* meetup have to link to event details page of the meetup (you can check events template how it is done there for inspiration).

regarding coverage, you dont need to worry about it we will accept you PR, coverage just say about how much of the code is covered by unit tests. We encourage to write some tests for the changes, byt it is not required for the PR to be accepted.

**Comment by neelab** [comment]
@ricco386 - Awesome, thanks. Can I do those other two issues (pagination and link) in a separate PR or do you prefer it in this one? Either way is fine, I just didn't know if you wanted to keep the refactoring separate.

**Comment by ricco386** [comment]
@neelab please do it in this PR as the request was part of the original issue #160

**Comment by neelab** [comment]
@ricco386 - Sure thing! 
Questions: 
- I'm not sure what you mean by styling the pagination - I don't see what is different in the event list view
- Also, I don't quite know how to show the event link. I'm guess it is similar to [https://github.com/pyconsk/django-konfera/blob/master/konfera/event/views.py#L45], but I see that in the meetup, the meetup details are being shown [https://github.com/pyconsk/django-konfera/pull/163/files#diff-f88798967ac8714a0517dfea05132a2eR12]. Any advice?

**Comment by ricco386** [comment]
Look at the bottom of the page: https://staging.pycon.sk/events/ and than if you look at the bottom of the meetups page (127.0.0.1:8000/meetups/ you have to have more than 5 meetups), you will notice that the buttons and page numbers looks differently. Please update yours to looks the same as on events page (use the same styles).

What I meant is that you open 127.0.0.1:8000/meetups/ in the browser you will see the list of all meetups, and when you click on meetup it should display the meetup details. So you have to update link in `<a href>` in https://github.com/neelab/django-konfera/blob/160-meetup-updates/konfera/templates/konfera/meetups.html template file (in both cases)

Hope this helps.

**Comment by neelab** [comment]
@ricco386 - Thanks! That was helpful. I didn't realize the templates folder was in there - I think Django is starting to make a little sense! 😄 
I added the new styles in my last commit.

**Comment by ricco386** [comment]
PR merged. Thanks @neelab

**Pull request merged**